src/PointVisual.cpp

```diff
@@ -38,14 +38,7 @@ PointVisual::~PointVisual() {
   scene_manager_->destroySceneNode(frame_node_);
 }
 
-void PointVisual::setPoint(const Ogre::Vector3 &point) {
-  Ogre::Vector3 scale(radius_, radius_, radius_);
-  point_->setScale(scale);
-
-  // Set the orientation of the arrow to match the direction of the acceleration
-  // vector.
-  point_->setPosition(point);
-}
+void PointVisual::setPoint(const Ogre::Vector3 &point) { point_->setPosition(point); }
```

Comment:
I moved the `setScale` command from out of here, down to `setRadius`, as that method better matches this functionality. Also, I was running into a frustrating issue where when resetting `PointVisual`s, I would call this method before `setScale`, and since the constructor for this class constructs with a radius of zero, I'd end up rendering a zero radius sphere.

Reply:
Also removed the comment, as it pertains to arrows, not points.

Reply:
Looks good!